tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -64,7 +64,7 @@ public async Task EachJobRunHasItsOwnScope()
 
         FakeTimer.Advance(TimeSpan.FromMinutes(1));
 
-        await Task.WhenAll(GetCompletionJobs(2));
+        await Task.WhenAll(GetCompletionJobs(2, CancellationToken));
```

Comment:
cf. https://xunit.net/xunit.analyzers/rules/xUnit1051

Reply:
Is `CancellationToken` coming from the new `TestContext`. Can't see it right now :D but given that it compiles, it should be correct

Reply:
Yes it does ;)

![image](https://github.com/user-attachments/assets/fc4abe3a-7741-483e-bccf-af0985f47245)